Vulnerabilities/xmlexternalentity.md

```diff
@@ -0,0 +1,24 @@
+# [XML External Entities](https://owasp.org/www-project-top-ten/2017/A1_2017-Injection)
+
+The XML External Entity vulnerability is when an xml file contains an external entity which is then expanded/evaluated by a parser.
```

Comment:
we should probably use `XML` (caps) in sentences to be consistent. That's what wiki does here: https://en.wikipedia.org/wiki/XML